workers/unity/Assets/Playground/Scripts/Cubes/ScoreCollision.cs

```diff
@@ -0,0 +1,59 @@
+using Generated.Playground;
+using Improbable.Gdk.Core;
+using Unity.Entities;
+using UnityEngine;
+
+[RemoveAtEndOfTick]
+public struct CollisionComponent : IComponentData
```

Comment:
Define this in the components directory relevant to this feature rather than with the monobehaviour.

Reply:
ACK